tools/its.py

```diff
@@ -468,11 +470,15 @@ def driver_book_appointment(self, driver, plz_impfzentrum):
             msg = "Termin erfolgreich gebucht!"
             self.log.success(msg)
             desktop_notification(operating_system=self.operating_system, title="Terminbuchung:", message=msg)
+            pushover_notification(self.notifications, msg)
+            telegram_notification(self.notifications, msg)
```

Comment:
Ich glaub wir wollen immer gleich alle drei Arten von Notification (Desktop, Pushover, Telegram); außer in einem Fall, wo bisher keine Desktop Notification ist. Aber da kann man ja auch eine Desktop Notification hinzufügen.

D. h. es wäre sinnvoller eine Methode zu schreiben, die direkt alle drei Arten von Notification mit der gleichen Nachricht auslöst. Die Methode sollte nicht-statisch sein, um Zugriff auf `self.operating_system` und `self.notifications` zu haben.

Reply:
Hab ich umgesetzt